generators/client/templates/angular/src/main/webapp/app/shared/language/_translate-partial-loader.provider.ts

```diff
@@ -31,8 +31,12 @@ export class TranslatePartialLoader implements TranslateLoader {
         return Observable.create(observer => {
             this.http.get(`${this.prefix}/${lang}/${part}${this.suffix}`).subscribe((res) => {
                 let responseObj = res.json();
-                Object.keys(responseObj).forEach(key => {
-                    combinedObject[key] = responseObj[key];
+                Object.keys(responseObj).forEach(key=>{
+                    if(!combinedObject[key]) {
+                        combinedObject[key] = responseObj[key];
+                    }else{
+                        $.extend(true,combinedObject[key], combinedObject[key], responseObj[key])
```

Comment:
We merge the observers, but the translations are in combinedObject. And  when we receive the response for the http request we overwrite the combinedObject[key] field. The problem is not when there are multiple files, but when they have the same key like :
File1 contains : { "test" : { a : 1}}
File 2 contains : { "test" : { b : 2}}

We want the merged object to be like this : { "test" : { a : 1, b : 2 }} am I right ?
And what we are doing right is now is just overwriting and we only get {test : {a : 1 }} or {test : { b :2 }} depending on which file is parsed last